feature-libs/product/bulk-pricing/components/bulk-pricing-table/bulk-pricing-table.component.spec.ts

```diff
@@ -1,15 +1,48 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { BulkPrice } from '../../core/model/bulk-price.model';
 
 import { BulkPricingTableComponent } from './bulk-pricing-table.component';
 
+import { RoutingService } from '@spartacus/core';
+import { BulkPricesService } from '../../core/services/bulk-prices.service';
+import { of } from 'rxjs';
+
+const mockState = {
+  state: {
+    params: {
+      productCode: 'testCode'
+    }
+  }
+}
+class MockRoutingService {
+  go() {}
+  getRouterState() {
+    return of(mockState);
+  }
+}
+
+class MockBulkPricesService {
```

Comment:
Please provide `implements Partial<service>` part to keep consistency with other unit tests